lib/jnpr/junos/rpcmeta.py

```diff
@@ -261,11 +261,10 @@ def load_config(self, contents, ignore_warning=False, **options):
 
         if contents is None and 'url' in options:
             pass
-        elif 'action' in options:
-            if options['action'] == 'set':
-                rpc.append(E('configuration-set', contents))
-            elif options['action'] == 'patch':
-                rpc.append(E('configuration-patch', contents))
+        elif 'action' in options and (options['action'] == 'set'):
+            rpc.append(E('configuration-set', contents))
+        elif 'action' in options and options['action'] == 'patch':
```

Comment:
Kindly follow  in below format 
elif ('action' in options) and (options['action'] == 'patch') :

Reply:
done